modules/installation-creating-aws-worker.adoc

```diff
@@ -158,22 +158,32 @@ $ aws cloudformation create-stack --stack-name <name> <1>
      --template-body file://<template>.yaml \ <2>
      --parameters file://<parameters>.json <3>
 ----
-<1> `<name>` is the name for the CloudFormation stack, such as `cluster-workers`.
+<1> `<name>` is the name for the CloudFormation stack, such as `cluster-worker-1`.
 You need the name of this stack if you remove the cluster.
 <2> `<template>` is the relative path to and name of the CloudFormation template
 YAML file that you saved.
 <3> `<parameters>` is the relative path to and name of the CloudFormation
 parameters JSON file.
++
+.Example output
+[source,terminal]
+----
+arn:aws:cloudformation:us-east-1:269333783861:stack/cluster-worker-1/729ee301-1c2a-11eb-348f-sd9888c65b59
+----
++
+[NOTE]
+====
+A stack that represents one worker node is created when the CloudFormation template is launched.
```

Comment:
See previous comment about active voice

Reply:
Thank you. I have changed the line to:

"The CloudFormation template creates a stack that represents one worker node."